### DISCUSSION TITLE
2024-03-27 Weekly Meeting 152 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#455] [Weekly Meetings]
Weekly Meeting 152 is happening on Wednesday, 27 March, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the NEW SERVER, in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/jVaUteJyh7).
We regularly use the # omi-meeting-chat channel for text chat.

Agenda
Check in, welcome
Fun finds!

Housekeeping:

last week's notes: https://github.com/omigroup/omigroup/discussions/451

**Comment by indiebio** [comment]
SoloSolipsist is checking this out: https://go.virtualrealitymarketing.com/ss-start

**Comment by indiebio** [comment]
GDC happened last week!

**Comment by mrmetaverse** [comment]
migrating the bots 
https://github.com/omigroup/omigroup/issues/443#issuecomment-1969467769

**Comment by mrmetaverse** [comment]
Keyframe will be speaking at AWE in Longbeach, and will be helping with other panels and the entertainment.

**Comment by indiebio** [comment]
Blog series idea: "Half-forgotten projects" - showcase things members tried, what they tried, what code is still available, what people would do if they got back to it. 
 - e.g. mrmetaverse's Masters' thesis that some plugin broke so then tried to recreate it in A-frame - https://xrshowcase.xyz/

**Comment by indiebio** [comment]
Consider merging with OMF to have a funding parent. - what would that look like, ideally for us? 
funding, open collective issues. https://fossfunding.com/
